src/osd/OSD.cc

```diff
@@ -9520,7 +9520,7 @@ void OSD::check_config()
   // some sanity checks
   if (cct->_conf->osd_map_cache_size <= (int)cct->_conf->osd_pg_epoch_persisted_max_stale + 2) {
     clog->warn() << "osd_map_cache_size (" << cct->_conf->osd_map_cache_size << ")"
-		 << " is not > osd_pg_epoch_persisted_max_stale ("
+		 << " is not > osd_pg_epoch_persisted_max_stale + 2 ("
```

Comment:
IMHO, this is even more confusing. it would read something like

> osd_map_cache_size (42) is not > osd_pg_epoch_persisted_max_stale + 2 (48)

so from an ordinary user's perspective, he/she will think that `osd_pg_epoch_persisted_max_stale` is `46` even if `osd_pg_epoch_persisted_max_stale` is actually 48.

how about also changing the following line like

```c++
<< cct->_conf->osd_pg_epoch_persisted_max_stale + 2 << ")";
```

Reply:
@tchaikov I totally agree that this can also confuse others. What do you think about `osd_pg_epoch_persisted_max_stale + 2 (48+2)` instead of `osd_pg_epoch_persisted_max_stale + 2 (50)`? It doesn't look very nice but should be more clear.

Reply:
@Fusl sorry for the latency. i agree with you that it's better.